k8s.io/README.md

```diff
@@ -9,6 +9,7 @@ Vanity URL(s)
 |  | k8s.io | kubernetes.io |
 | --- | --- | --- |
 | APT downloads| https://apt.k8s.io | https://apt.kubernetes.io |
+| Blog | https://blog.k8s.io | |
```

Comment:
I'm not sure why this table has both `k8s.io` and `kubernetes.io` for each, but we should probably include `blog.kubernetes.io` and `sigs.kubernetes.io` as well.